makefiles/pseudomodules.inc.mk

```diff
@@ -463,6 +463,7 @@ PSEUDOMODULES += shell_cmd_udptty
 PSEUDOMODULES += shell_cmd_vfs
 PSEUDOMODULES += shell_cmds_default
 PSEUDOMODULES += shell_hooks
+PSEUDOMODULES += shell_json
```

Comment:
Shouldn't this be rather `shell_cmd_cmds_json` to match the pattern of other shell command modules?

Reply:
I think that can be confusing, as the others are all externally implemented shell commands (they have a description and show up in `help`). This is however implemented as a builtin (see below for why) and behaves differently.